environments/icds-cas/fab-settings.yml

```diff
@@ -1,5 +1,5 @@
 acceptable_maintenance_window:
-  hour_start: 0
-  hour_end: 7
+  hour_start: 20
+  hour_end: 23
```

Comment:
A deploy at 11pm won't be likely to end before 1130pm. Can we make this `22:30`?